### ISSUE TITLE
This is a 3D sim

### ISSUE THREAD

**Issue by andrewadare** [phetsims/capacitor-lab-basics#157]
The sim uses a custom 3D model-view transform. Several components have 3D geometry, but just as many don't. This is causing maintenance problems, readability problems, and phet-io problems like https://github.com/phetsims/phet-io/issues/811.

It's leading to code like this in CircuitSwitch,
```js
   /**
     * Get the location of the endpoint for the circuit switch segment.
     *
     * @return {Vector2} [description]
     */
    getSwitchEndPoint: function() {

      var endPoint = this.switchSegment.endPointProperty.value;

      // assert && assert( endPoint instanceof Vector2 );

      // Sometimes Vector3s are getting serialized as Vector2, see https://github.com/phetsims/phet-io/issues/811
      // So until WireSegment supports single types (whether Vector2 or Vector3), we use this hack to support both types
      // TODO: Remove this logic after WireSegment is sorted out.
      if ( endPoint instanceof Vector2 ) {
        return endPoint.copy();
      }
      else if ( endPoint instanceof Vector3 ) {
        return endPoint.toVector2();
      }
      else {
        throw new Error( 'bad type' );
      }
    },
```
and this snippet from WireShapeCreator:
```js

       if ( segment.endPointProperty.value instanceof Vector2 ) {
         wireShape.moveToPoint( startPoint ).lineToPoint( endPoint );
       }
       else {
         wireShape.moveToPoint( startPoint ).lineToPoint( endPoint.toVector2() );
       }
  
```
WireSegment itself takes a startPoint and an endPoint vector, and at runtime, sometimes it's getting a Vector2, sometimes it's a Vector3. In addition to ubiquitous `.toVector2()` calls, there's a lot of duck-typing going on, both in the sim and in common code, that allows this to work. But if there's ever a port to a full Capacitor Lab sim, the confusion and inconsistency are sure to cause problems.

I think the only robust solution is to fully commit to 3D geometry in the model, get rid of the conversions and conditional code, and only convert to 2D when needed for the view.

**Comment by andrewadare** [comment]
This is looking pretty good now. Local fuzz tests pass, and no more ambiguity over whether positions are 2D or 3D in the model. @phet-steele I'm going to leave this issue open, and please comment here if you spot any problems that look related to this in the next testing round.

**Comment by jonathanolson** [comment]
Try adding the assertion:
```js
assert && assert( modelPoint instanceof Vector3 );
```
at the top of CLBModelViewTransform3D's modelToViewPosition (it expects Vector3s). It will be fired from:
```js
    voltmeter.bodyLocationProperty.link( function( bodyLocation ) {
      self.translation = modelViewTransform.modelToViewPosition( bodyLocation );
    } );
```
in VoltmeterBodyNode, triggered by passing in what was presumably now a Property(Vector3) to a MovableDragHandler which sets location Vector2-style.

Didn't fully track down to determine origin/fix. Detected by Scenery type assertions being added in a non-master branch (that will presumably be master soon).

**Comment by andrewadare** [comment]
@jonathanolson thanks for catching this. It should be addressed by 75665ef.

**Comment by andrewadare** [comment]
Closing since no issues came up in dev testing with regard to this.
